ftdc_tools/ftdc_decoder.py

```diff
@@ -0,0 +1,301 @@
+import collections
```

Comment:
I wasn't on the original review, so I'm assuming most of this was unchanged and I shouldn't look at it? Please lmk if that's wrong and I can give it a review as well.